cpp/zotaut/journal_timeliness_checker.cc

```diff
@@ -91,9 +94,10 @@ int Main(int argc, char *argv[]) {
         if (section.find("user_agent") != section.end())
             continue; // Not a journal section.
 
-        const auto delivery_mode(section.getEnum("zotero_delivery_mode", ZoteroHarvester::Config::STRING_TO_UPLOAD_OPERATION_MAP,
-                                                 ZoteroHarvester::Config::UploadOperation::NONE));
-        if (delivery_mode != ZoteroHarvester::Config::UploadOperation::LIVE or section.getBool("zeder_newly_synced_entry", false))
+        const auto delivery_mode(static_cast<ZoteroHarvester::Config::UploadOperation>(
+             section.getEnum("zotero_delivery_mode", ZoteroHarvester::Config::STRING_TO_UPLOAD_OPERATION_MAP,
```

Comment:
Normally, zotaut ini keys should not be referenced directly with a string; instead ZoteroHarvester::Config::GetIniKeyString should be used. However a) this is very complicated and i've never been happy with that solution either and b) there are already other parts of this file that use keys directly, so i guess we just leave it as-is for the moment before we open up another big construction site for a small change like this...

Reply:
The approach of determining the keys is identical to the one already present beforehand. Only the type conversion with the `enum` had to be newly introduced to enable passing the result to a function.